app/models/weets.js

```diff
@@ -0,0 +1,30 @@
+module.exports = (sequelize, DataTypes) => {
+  const Weet = sequelize.define(
+    'Weet',
+    {
+      id: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        unique: true,
+        autoIncrement: true,
+        primaryKey: true
+      },
+      content: {
+        type: DataTypes.STRING,
+        allowNull: false
+      }
+    },
+    {
+      underscored: true,
+      timestamps: true,
+      paranoid: true,
+      tableName: 'weets'
+    }
+  );
+
+  Weet.associate = models => {
+    Weet.belongsTo(models.User);
```

Comment:
Weet.belongsTo(models.User, { as: 'userCreator', foreignKey: 'creatorId' });
So, when you get weet info, the property creatorId will be present